addons/website_slides/models/slide_slide.py

```diff
@@ -193,8 +193,7 @@ class Slide(models.Model):
         help="Subtype of the slide category, allows more precision on the actual file type / source type.")
     document_google_url = fields.Char('Document Link', related='url', readonly=False,
         help="Link of the document (we currently only support Google Drive as source)")
-    document_binary_content = fields.Binary('PDF Content', related='binary_content', readonly=False,
-        help="Used to filter file input to PDF only")
+    document_binary_content = fields.Binary('PDF Content', related='binary_content', readonly=False)
```

Comment:
How does that filter work? The idea is to make sure of whether this help should be converted to a code comment or if it is really not necessary at any level.